src/moveFiles.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function normalizePaths(source, destination) {
+  const currentPath = path.join(__dirname, source);
+  let newPath = '';
+
+  if (destination.endsWith('/')) {
+    const dirPath = path.join(__dirname, destination);
+
+    newPath = path.join(dirPath, source);
+  } else {
+    if (destination.includes('./')) {
```

Comment:
Use helper function for such nesting conditions
![image](https://github.com/mate-academy/node_move-files/assets/70009205/266d063f-b499-404f-993a-3efebf22e7aa)